adminSiteClient/gdocsDeploy.ts

```diff
@@ -0,0 +1,62 @@
+import {
+    OwidArticleContent,
+    OwidArticleType,
+} from "../clientUtils/owidTypes.js"
+import { isEqual, omit } from "../clientUtils/Util.js"
+
+export const checkFullDeployFallback = (
+    prevGdoc: OwidArticleType,
+    nextGdoc: OwidArticleType,
+    hasChanges: boolean
+) => {
+    return hasChanges && (prevGdoc.published || nextGdoc.published)
+}
+
+/**
+ * This function checks if the article has changed in a way that is compatible
+ * with a lightning update. It does this by excluding certain props ("lightning
+ * props") from the comparison. If nothing has changed except for the lightning
+ * props, then it is safe to do a lightning update.
+ *
+ */
+export const checkLightningUpdate = (
```

Comment:
Ok. I'm changing to a slightly shorter version to avoid weird line wraps which make reading the code a bit more difficult. Hopefully the JSDoc on hover helps clarifying without too much friction if necessary.